.github/pull_request_template.md

```diff
@@ -13,6 +14,20 @@ _['low', 'medium', 'high', etc.]_
 1. _[First testing step]_
 2. ...
 
-## PR Checklist
-- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
-- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
+## Backend / Functional Validation
+### Code
+- [ ] Are the naming conventions following our standards?
+- [ ] Are PHP functions and variables in `snake_case` and not `camelCase`?
```

Comment:
Maybe change "PHP functions and variables" to "non-class file (e.g., .module) functions and variables ."  That's distinguishes it from class methods in my mind.

Reply:
I can see that. I was just using the language from the Drupal PHP coding standards page: https://www.drupal.org/docs/develop/standards/php/php-coding-standards#s-functions-and-variables

Reply:
Lol! I did too. I copied and pasted that phrase from that page. Albeit another section.

I wonder if you could get away with using PHP Code Sniffer instead of having the human review for coding standards. I have this alias in my bashrc file: `alias drupalcs="phpcs --standard=Drupal --extensions='php,module,inc,install,test,profile,theme,js,css,info,txt'"`

Then I just run it against any code I've updated. 

Maybe a discussion for dev team?

Reply:
This can all be standardized with a `phpcs.xml` file in the project root.   Here's an example from another project

```xml
<?xml version="1.0"?>
<ruleset name="Project Standard" namespace="THIS_PROJECT_NAME\project\CS\Standard">
<arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,md,yml"/>
<rule ref="vendor/drupal/coder/coder_sniffer/Drupal">
<!--  If there's something from that set that we don't want, it can be excluded. -->
<!--
<exclude name="Drupal.Commenting.FunctionComment.MissingReturnComment"/>
<exclude name="Drupal.Commenting.DocComment.MissingShort"/>
<exclude name="Drupal.Commenting.FunctionComment.MissingParamComment"/>
<exclude name="Drupal.Files.LineLength.TooLong"/>
-->
</rule>
<!--  This one is a bit extreme.  Might not want to start with this -->
<rule ref="vendor/drupal/coder/coder_sniffer/DrupalPractice" />

<file>web/modules/custom</file>
<file>web/themes/custom</file>

<!--  Exclude 3rd-party libraries and compiled code. -->
<exclude-pattern>*/node_modules/*</exclude-pattern>
<exclude-pattern>emulsify.php</exclude-pattern>
<exclude-pattern>*/themes/custom/*/*.md</exclude-pattern>
<exclude-pattern>*/themes/custom/*/.out</exclude-pattern>
<exclude-pattern>*/themes/custom/*/dist</exclude-pattern>

</ruleset>
```

Reply:
Probably could be added to CI/CD pipeline as well.

Reply:
Have set this up in a separate PR: https://github.com/SU-HSDO/suhumsci/pull/1686